picard/coverart/image.py

```diff
@@ -178,17 +180,6 @@ def __init__(self, url=None, types=None, comment='', data=None, support_types=No
         if data is not None:
             self.set_data(data)
 
-    def parse_url(self, url):
```

Comment:
https://github.com/metabrainz/picard-plugins/blob/c03c7f1fdb2546d4e3467d99859406ce6cf0bc05/plugins/theaudiodb/__init__.py#L97 says it was for Picard < 2.1.3

Reply:
Yes exactly. My point was that nothing breaks due to this, we can keep that method override. With old code it will work (and even fix the bug for old Picard versions), for the new code it just never gets called.